default.nix

```diff
@@ -111,12 +108,12 @@ pkgs.haskell.packages.${compiler}.developPackage {
     base64-bytestring = "1.0.0.3";
   };
 
-  modifier = drv: pkgs.haskell.lib.overrideCabal drv (attrs: {
+  modifier = drv: onlyStatic (pkgs.haskell.lib.overrideCabal drv (attrs: {
```

Comment:
Instead of baking the static executable treatment into the modifier here, I suggest something like this: https://github.com/kadena-io/chainweb-data/commit/4e20c20886166b972de2df66d3349f59f1dd5955 (You can `cherry-pick` it if that makes sense)